polynote-kernel/src/main/scala/polynote/kernel/PolyKernel.scala

```diff
@@ -57,7 +57,7 @@ class PolyKernel private[kernel] (
   /**
     * The task queue, which tracks currently running and queued kernel tasks (i.e. cells to run)
     */
-  protected lazy val taskQueue: TaskQueue = TaskQueue(statusUpdates).unsafeRunSync()
+  protected lazy val taskQueue: TaskManager = TaskManager(statusUpdates).unsafeRunSync()
```

Comment:
maybe rename `taskQueue` to `taskManager` as well?

Reply:
:+1: done